spec/models/note_spec.rb

```diff
@@ -5,9 +5,9 @@
 
 RSpec.describe Note do
   before(:example) do
-    Setting['advanced.versions'] = 'true'
-    Setting['advanced.version_gap_distance'] = 10
-    Setting['advanced.version_gap_minutes'] = 60
+    ENV['versions'] = 'true'
+    ENV['version_gap_distance'] = '10'
+    ENV['version_gap_minutes'] = '60'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.